libr/core/cmd_print.c

```diff
@@ -4507,6 +4507,49 @@ R_API void r_print_code(RPrint *p, ut64 addr, const ut8 *buf, int len, char lang
 	}
 }
 
+static void get_string_section(RCore *core, const char** section_name, ut64* address) {
```

Comment:
wtf we have 3 functions that do the same. see r_core_get_section_name

Reply:
Oh. I just refactored. I'll remove it and use that one you suggested